outlines/models/hf_transformers.py

```diff
@@ -93,7 +93,7 @@ def call_model_generate_method(
     prompt = list(prompt)
 
     tokenizer = AutoTokenizer.from_pretrained(model_name, padding_size="left")
-    model = AutoModelForCausalLM.from_pretrained(model_name)
+    model = AutoModelForCausalLM.from_pretrained(model_name, trust_remote_code=True)
```

Comment:
We probably shouldn't set this to `True` by default

Reply:
This 100% shouldn't be set to `True` by default. As Rémi said, we want to respect the library defaults here.